yarn-project/boxes/blank/src/contracts/Nargo.toml

```diff
@@ -5,4 +5,4 @@ compiler_version = "0.1"
 type = "contract"
 
 [dependencies]
-aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="master", directory="yarn-project/aztec-nr/aztec" }
+aztec = { path = "../../../../aztec-nr/aztec" }
```

Comment:
same as above - unsure if this will still work if someone npm installs our box

Reply:
they dont get npm installed actually, its actually just copying the subpackage from github and doing some version replacements/overrides.  but similar issue, not having the full monorepo available